sdk/servicebus/service-bus/src/receivers/sessionReceiver.ts

```diff
@@ -141,6 +140,43 @@ export class SessionReceiverImpl<ReceivedMessageT extends ReceivedMessage | Rece
         throw error;
       }
     }
+
+    // `createInitializedSessionReceiver` will set this value by calling init()
+    // so we just temporarily set it to "" so we can get away with it never being
+    // `undefined`.
+    this.sessionId = "";
```

Comment:
This makes me slightly nervous... Turns out empty string is a valid session id...

Reply:
Yes, it's part of the reason why I made it so it's not possible to construct a SessionReceiver directly. The window of time here is between constructor() and init() which is small and manageable.